### PR TITLE
[FIX] public_budget: Se muestra en en la vista de trasnferencias el campo Fecha

### PR DESCRIPTION
**Pull request by mem-adhoc**
Ticket: 59996

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/476).

File: public_budget/views/account_payment_views.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
             <field name="date" position="attributes">
                 <!-- <attribute name="invisible">1</attribute> -->
                 <!-- al final lo hacemos invisible así porque en transferencias lo queremos visible -->
-                <attribute name="attrs">{'readonly': [('state', '!=', 'draft')], 'invisible': [('payment_type', '!=', 'transfer')], 'required': [('payment_type', '=', 'transfer')]}</attribute>
+                <attribute name="attrs">{'readonly': [('state', '!=', 'draft')], 'required': [('payment_type', '=', 'transfer')]}</attribute>
```
Comment (pablohmontenegro) [inline public_budget/views/account_payment_views.xml]: @mem-adhoc hi Maxi, please delete ", 'required': [('payment_type', '=', 'transfer')]"

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-public-administration#477

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#477

**Pull request closed without merging**